### PR TITLE
0.7.0 release candidate 1

### PR DESCRIPTION
**Pull request by praveksharma**
Please see the relevant pre-release (currently in draft):  https://github.com/open-quantum-safe/oqs-provider/releases/tag/untagged-af15abfacccde382a3d8

This updates the version string, the release notes, and points CI to liboqs 0.11.0.

Checks done:

- Stable under `generate.py`.
- CI is green.
- 'LIBOQS_BRANCH=0.11.0 ./scripts/fullbuild.sh -F && ./scripts/runtests.sh -V` runs OK.
- Passing `release-tests.sh` with liboqs 0.11.0 and openssl 3.3.2.
- nginx, curl, and httpd docker images building OK. 

This PR will not land; once approved, a new 0.7.0-release will be created which will be merged.

### CODE REVIEW

**Review by baentsch — APPROVED** [review]
LGTM. Thanks very much for putting this together @praveksharma ! Only question pertains to rationale for fixing certain `openssl` version references (why move from 3.1 to 3.2; why not to latest in respective branch?)

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+
+### What's New
+
+In addition to updating documentation, improving the CI, and fixing issues uncovered by compiler warnings and static analysis, this release of oqs-provider:
+
+* Adds support for MAYO from Round 1 of [NIST’s Post-Quantum Signature On-Ramp process](https://csrc.nist.gov/projects/pqc-dig-sig/round-1-additional-signatures).
+* Adds support for CROSS from Round 1 of [NIST’s Post-Quantum Signature On-Ramp process](https://csrc.nist.gov/projects/pqc-dig-sig/round-1-additional-signatures).
+* Updates ML-KEM's code points in line with internet draft [draft-kwiatkowski-tls-ecdhe-mlkem-02](https://www.ietf.org/archive/id/draft-kwiatkowski-tls-ecdhe-mlkem-02.html).
+* Updates the `fullbuild.sh` build script to build against liboqs with formally verified Kyber-512 and Kyber-768 from [libjade](https://github.com/formosa-crypto/libjade) turned on by default; see `OQS_LIBJADE_BUILD` under `CONFIGURE.md` for more information.
+* Reverses keyshares for X25519MLKEM768, SecP256r1MLKEM768, and X448-ML-KEM-768 TLS hybrids in line with [draft-kwiatkowski-tls-ecdhe-mlkem-02](https://www.ietf.org/archive/id/draft-kwiatkowski-tls-ecdhe-mlkem-02.html).
```
Comment (ghen2) [inline RELEASE.md:31]: Only the X25519 and X448 hybrids have been reversed ?

Comment (baentsch) [reply RELEASE.md:31]: I'd hope so. Thanks for the catch @ghen2 . @praveksharma please remove P256 from the list.

Comment (praveksharma) [reply RELEASE.md]: This has been fixed, thank you @ghen2.

**Comment by praveksharma** [comment]
> Only question pertains to rationale for fixing certain `openssl` version references (why move from 3.1 to 3.2; why not to latest in respective branch?)

This was an oversight, thank you for pointing this out @baentsch. I have changed this to openssl 3.3.2 which is the latest release.

The draft pre-release has been published. The PR for the release, #534, is up for review and includes the change suggested here.

**Pull request merged**